webui/src/pages/repositories/repository/settings/retention.jsx

```diff
@@ -0,0 +1,187 @@
+    else {
+        content = (
+                    <>
+                        <ActionsBar>
+                            <ActionGroup><h4>Garbage Collection Policy</h4></ActionGroup>
+                            <ActionGroup orientation="right">
+                                <RefreshButton onClick={doRefresh}/>
+                                <Button variant="success" onClick={() => setShowCreate(true)}>Edit Policy</Button>
+                            </ActionGroup>
+                        </ActionsBar>
+                        <ActionsBar>
+                            <ActionGroup orientation="right">
+                                <ToggleSwitch label={"JSON view"} id={"policy-json-switch"} onChange={setJsonView}
+                                              defaultChecked={jsonView}/>
+                            </ActionGroup>
+                        </ActionsBar>
+                        {jsonView ? (
+                            <pre className={"policy-body"}>{JSON.stringify(policy, null, 4)}</pre>
+                        ) : (
+                            <>
+                                <Table borderless>
+                                    <tbody>
+                                    <tr key={'branch-default'}>
+                                        <td><code>Default retention days: {policy.default_retention_days}</code></td>
+                                    </tr>
+                                    </tbody>
+                                </Table>
+                                <Card>
+                                    <Table>
+                                        <thead>
+                                        <tr>
+                                            <th width={"80%"}>Branch</th>
+                                            <th>Retention Days</th>
+                                        </tr>
+                                        </thead>
+                                        <tbody>
+                                        {policy.branches.map((branch, i) => {
+                                            return (
+                                                <tr key={`branch-${i}`}>
+                                                    <td><code>{branch.branch_id}</code></td>
+                                                    <td><code>{branch.retention_days}</code></td>
+                                                </tr>
+                                            );
+                                        })}
+                                        </tbody>
+                                    </Table>
+                                </Card>
+                                <br/>
+                                <br/>
+                                <Row className="pt-2 ml-2">
+                                    For information about how to run the garbage collection see the&nbsp;<a
```

Comment:
Suggestion for the text:

This policy determines for how long objects are kept in the storage after they are deleted in lakeFS. For more information, see the docs.